src/main/java/racingcar/RacingService.java

```diff
@@ -26,4 +33,28 @@ public void move(Racing racing) {
     private int generateRand() {
         return new Random().nextInt(10);
     }
+
+    public Set<String> selectWinner(Racing racing) {
+        Set<String> winners = new HashSet<>();
+        int max = 0;
+        for (Car car : racing.getCars()) {
+            if (car.getCurrentDistance() > max) {
```

Comment:
```suggestion
            if (car.큼니까?(max)) {
```

이렇게 된다면 단위테스트하기에도 더 좋을것 같아요!
객체에게 메시지를 전달해보는 연습을 해보아요~